README.md

```diff
@@ -19,6 +19,10 @@ Phoenix.js documentation is available at [https://hexdocs.pm/phoenix/js](https:/
 
 We appreciate any contribution to Phoenix. Check our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [CONTRIBUTING.md](CONTRIBUTING.md) guides for more information. We usually keep a list of features and bugs [in the issue tracker][4].
 
+### Installing the Phoenix (phx_new Mix archive)
+
+You can install stable/released versions of Phoenix following the instructions [here](https://hexdocs.pm/phoenix/installation.html#phoenix)
```

Comment:
I don't think this belongs here, since it isn't relevant to contributing. I think it should either be mentioned in the "Getting started" section.

>  ## Getting started
>  	 
>   See the official site at http://www.phoenixframework.org/
>
>   Install the latest version of Phoenix by following the instructions at https://hexdocs.pm/phoenix/installation.html#phoenix